src/content/1.7/modules/creation/module-translation/new-system.md

```diff
@@ -214,25 +230,15 @@ For instance, translating the string "Welcome to this page!" can be done like th
 {l s='Welcome to this page!' d='Modules.Mymodule.Somefile'}
 ```
 
-#### Twig files
-
-Wordings in Twig .twig files can be translated using the `trans` filter. It works similarly as the `trans()` method described for PHP files above:
-
-```twig
-{# file: something.twig #}
-
-{{ 'Welcome to this page!'|trans({}, 'Modules.Mymodule.Admin') }}
-```
-
-The first parameter can be used to replace tokens in your wording after it's translated:
+You can replace placeholders in your translated wordings using the `sprintf` parameter:
 
-```twig
-{{ 'Hello %username%!'|trans({'%username%': 'John'}, 'Modules.Mymodule.Admin') }}
+```smarty
+{l s='Hello %username%!' sprintf=['%username%' => 'John'] d='Modules.Mymodule.Somefile'} }}
 ```
 
 ## Translating your module
 
-To be translatable through the new translation interface, modules must opt-in. This can be done by declaring this function on your module's main class:
+Modules must opt-in to become translatable using the new Back Office translation interface. This can be done by declaring the following function on your module's main class:
```

Comment:
We dont look at this opt-in when requesting the modules in the "Modify translations" form

Reply:
This toggles access to the new translation interface. If the module hasn't opted in, you get the legacy translation interface ([see TranslationRouteFinder](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/Adapter/Translations/TranslationRouteFinder.php#L130))

Reply:
Ok. It's done on the routing level. Nothing to do when feeding the translator ?